test/plugin/containers/jvm-container/docker/run.sh

```diff
@@ -76,11 +76,11 @@ healthCheck http://localhost:12800/status
 healthCheck ${SCENARIO_HEALTH_CHECK_URL}
 
 echo "To visit entry service"
-curl -s --max-time 3 ${SCENARIO_ENTRY_SERVICE}
+curl -s --max-time 60 ${SCENARIO_ENTRY_SERVICE}
```

Comment:
If access too many, the mock collector may be OOM.

Reply:
this should be timeout, not retry count

Reply:
It is still a single thread to visit the mok-collector without retry. Increasing the timeout time why it could improve the success rate. Make tests more stable.
There is no retry but will timeout in 60s. I think this change is necessary.